crates/hyperqueue/src/client/output/cli.rs

```diff
@@ -888,9 +893,32 @@ pub fn format_task_duration(
             start_date,
             end_date,
             ..
-        } => *end_date - *start_date,
+        } => (Option::Some(*start_date), Option::Some(*end_date))
+    }
+}
+
+pub fn format_task_time(
+    date: Option<DateTime<Utc>>
+) -> String
+{
+    return match date {
+        None => "".to_string(),
+        Some(d) => d.to_string(),
+    };
+}
+
+pub fn format_task_duration(
+    start: Option<DateTime<Utc>>,
+    end: Option<DateTime<Utc>>,
+    completion_date_or_now: &chrono::DateTime<chrono::Utc>,
+) -> String
+{
+    return match (start, end) {
+        (None, None) => "".to_string(),
+        (Some(start), None) => human_duration(*completion_date_or_now - start),
```

Comment:
`completion_date_or_now` is not needed anymore, just replace it with the current time and remove `completion_date_or_now` entirely from the output API.